tgstation.dme

```diff
@@ -1352,6 +1352,7 @@
 #include "code\modules\mob\living\simple_animal\shade.dm"
 #include "code\modules\mob\living\simple_animal\simple_animal.dm"
 #include "code\modules\mob\living\simple_animal\spawner.dm"
+#include "code\modules\mob\living\simple_animal\status_procs.dm"

```

Comment:
But most of those mobs dont have proper handling for those statuses anyways? Unless its more shitcode on mob/living.